scripts/mppc_to_multiprobe.py

```diff
@@ -0,0 +1,298 @@
+    # shape of full mppc is (8, 8)
+    cell_size_y = int(image.shape[0] / 8)
+    cell_size_x = int(image.shape[1] / 8)
+    image_A1 = image[0:cell_size_y, 0:cell_size_x]
+    image_H1 = image[0:cell_size_y, -cell_size_x:]
+    image_A8 = image[-cell_size_y:, 0:cell_size_x]
+    image_H8 = image[-cell_size_y:, -cell_size_x:]
+    image_E4 = image[cell_size_y*3:cell_size_y*4, cell_size_x*4:cell_size_x*5]
+
+    images = [image_A1, image_A8, image_H1, image_H8, image_E4]
+
+    # thresholding
+    # Known:
+    #   - size of single image (800px x 800px => 30um x 30um = 900 um2)
+    #   - size of active mppc cell image (bright square) approximately the same at all times
+    #     (3um x 3um = 9um2 => 1% of image)
+    # set 99% of the pixels with the lowest values to 0, set the rest of the pixels to 1
+    for num_img, img in enumerate(images):
+        shape = img.shape
+        img_histogram = numpy.bincount(numpy.ravel(img))
+        index_array = numpy.cumsum(img_histogram) >= 0.99 * shape[0] * shape[1]  # bool array: True if 99% accumulated
+        threshold = index_array.argmax()  # find the first occurrence of True
+        img[img <= threshold] = 0  # set 99% of px with the lowest values to 0
+        img[img > threshold] = 1  # set the remaining 1% to 1
+
+        kernel = cv2.getStructuringElement(cv2.MORPH_RECT, (3, 3))
+        # get rid of noise
+        img_open = cv2.morphologyEx(img, cv2.MORPH_OPEN, kernel, iterations=1)
+        # fill holes in bright square
+        img_close = cv2.morphologyEx(img_open, cv2.MORPH_CLOSE, kernel, iterations=2)
+        images[num_img] = img_close
+
+        # find connected component in thresholded image
+        img_cc, num_labels = skimage.measure.label(img_close, background=0, connectivity=1, return_num=True)
+
+        # there should be only one label in num_labels
+        if num_labels != 1:
+            logging.info("There should be only one active area in the image. However, found {:f} labelled"
+                         "objects.".format(num_labels))
+        # Labelled object should have the size of about 1% of total image in px. Otherwise active area cut of or
+        # outside of the scanning area e.g. due to too much rotation of the mppc.
+        # Use a threshold of 90% of the expected size to allow for variation due to dilation/erosion operation.
+        active_area_found = False
+        # find the connected component
+        # Note: num_labels does not include any background values
+        for label in range(num_labels):
+            # check that the size of the connected component is at least 10% of the size
+            # of the active area (about 1% of the total area)
+            # If not, it is either noise or only part of the active area was captured due to too much rotation
+            # (active area a edge and thus cut-off).
+            if len(img_cc[img_cc == label + 1]) < 0.1 * 0.01 * cell_size_y * cell_size_x:  # as 0 is background
```

Comment:
Maybe this is also more explicit:
`len(img_cc[img_cc == label + 1])` -> `numpy.count_nonzero(img_cc == label + 1)`